test/verifiedTests/testConnectedComponents/testConnectedComponents.m

```diff
@@ -0,0 +1,52 @@
+% The COBRAToolbox: testConnectedComponents.m
+%
+% Purpose:
+%     - tests the Connected Components functionality using a toy model
+%
+%
+% Authors:
+%     - Thomas Pfau
+
+%% The model created looks as follows:
+%
+%
+%   <-> A -> B -> C <->
+%
+%
+%              -> F -
+%             /      \
+%   <-> D -> E -> G ---> H <->
+%            ^
+%            |
+%            v
+%           
+% And should yield 2 connected components, along with a bunch of exchangers
+% which are ignored.
+cwd = pwd;
```

Comment:
May you use the same nomenclature as in every other test? That will help us for future maintenance ...